stack.yaml

```diff
@@ -1 +1,6 @@
 resolver: lts-14.20
+
+nix:
+  enable: true
+  packages:
+    - zlib
```

Comment:
Since i'm on NixOS this was needed in order to run a nix shell for the project ( zlib is a native dependency ). It could be a convenience for Nix/NixOS users but i'm happy to remove it if you consider it not needed.

Reply:
@Nimor111 I'm okay with this if it doesn't interfere with normal users and doesn't require `stack` users to have `nix`. I'm not using Nix personally.

Reply:
@chshersh I'll double check if it works without nix and get back on this.

Reply:
It does require nix, i'll remove them.